Touchpose/Touchpose/QTouchposeApplication.m

```diff
@@ -303,14 +303,17 @@ - (void)setShowTouches:(BOOL)showTouches
 {
     if (showTouches)
     {
-        if (_touchView == nil && self.keyWindow)
+        for (UIWindow *window in [UIApplication sharedApplication].windows)
```

Comment:
Can you use `self` instead of `[UIApplication sharedApplication]`? (is it the same object)